### PR TITLE
Added docs on forthcoming changes to listbox selection.

### PR DESCRIPTION
**Pull request by grokys**
And added a section about styling containers.

Contains documentation for pre-release functionality: do not merge until 0.10 is released.

### CODE REVIEW

File: wwwroot/docs/controls/listbox.md [hunk 1]
```diff
@@ -26,6 +26,152 @@ each item inside a red border with rounded corners:
...
+By default bindings to this property are two-way.
+
+:::note
+Do not bind to this property if your `Items` collection contains duplicates as it is impossible to
+distinguish between duplicate values.
+:::
+
+## Selection
+
+The `Selection` property exposes an `ISelectionModel` object which exposes various methods to track
```
Comment (Kryptos-FR) [inline wwwroot/docs/controls/listbox.md]: Repetition of "exposes". Could be rephrased as, for example:

> The `Selection` property exposes an `ISelectionModel` object with various methods to track

File: wwwroot/docs/controls/listbox.md [hunk 2]
```diff
@@ -26,6 +26,152 @@ each item inside a red border with rounded corners:
...
+:::
+
+`SelectionModel` also exposes batching functionality through its `Update()` method and a
+`SelectionChanged` event which details exactly which items have been selected and deselected.
+
+```xml
+<ListBox Items="{Binding Items} Selection="{Binding Selection}">
+```
+
+```chsarp
```
Comment (Kryptos-FR) [inline wwwroot/docs/controls/listbox.md]: Typo `csharp`

Comment (grokys) [reply wwwroot/docs/controls/listbox.md]: Thanks for the review @Kryptos-FR - fixed those and added some more example code.

**Comment by grokys** [comment]
Closing: info added to #111.

**Pull request closed without merging**